packages/reqresp/src/types.ts

```diff
@@ -97,4 +97,5 @@ export interface TypeSerializer<T> {
   deserialize(bytes: Uint8Array): T;
   maxSize: number;
   minSize: number;
+  equals(a: T, b: T): boolean;
```

Comment:
Why do you need equals?
- If you are using this for tests only do not add unnecessary stuff on src code
- If you are using this for test assertion, it's really bad to debug. Always compare actual data, assert serialized == serialized or json == json, else the failed assertion will be "expected true, got false" which does not give any data point to debug

Reply:
It was added to re-use existing helper `isEqualSszType`

Reply:
If `isEqualSszType` is used only on tests, please do not add here